chips/nrf51/src/lib.rs

```diff
@@ -24,3 +24,4 @@ pub mod pinmux;
 pub use chip::NRF51;
 pub mod temperature;
 pub mod trng;
+pub mod radio;
```

Comment:
FYI, while I wanted it changed in `main.rs`, I'm generally pretty fine with the peripheral being called `radio` in the `chips` crate since the peripheral in the nRF manual is actually called `RADIO`.